packages/rest/src/rest.server.ts

```diff
@@ -227,32 +231,25 @@ export class RestServer extends Context implements Server, HttpServerLike {
     }
   }
 
-  protected _handleHttpRequest(
-    request: Request,
-    response: Response,
-    options: RestServerConfig,
-  ) {
-    if (
-      request.method === 'GET' &&
-      request.url &&
-      request.url in OPENAPI_SPEC_MAPPING
-    ) {
+  protected _handleHttpRequest(request: Request, response: Response) {
+    const mapping = this.config.openApiSpec!.endpointMapping!;
+    if (request.method === 'GET' && request.url && request.url in mapping) {
       // NOTE(bajtos) Regular routes are handled through Sequence.
       // IMO, this built-in endpoint should not run through a Sequence,
       // because it's not part of the application API itself.
       // E.g. if the app implements access/audit logs, I don't want
       // this endpoint to trigger a log entry. If the server implements
       // content-negotiation to support XML clients, I don't want the OpenAPI
       // spec to be converted into an XML response.
-      const settings = OPENAPI_SPEC_MAPPING[request.url];
-      return this._serveOpenApiSpec(request, response, settings);
+      const form = mapping[request.url];
```

Comment:
Why is this called `form`? Seems unintuitive.

Reply:
What's a better name for `version + format`?